etc/nextcloudpi-config.d/letsencrypt.sh

```diff
@@ -82,6 +90,10 @@ EOF
     # delayed in bg so it does not kill the connection, and we get AJAX response
     bash -c "sleep 2 && service apache2 reload" &>/dev/null &
     rm -rf $NCDIR/.well-known
+    
+    # update configuration
+    /usr/local/sbin/update-rc.d letsencrypt enable
```

Comment:
a little change I made

```
[[ "$DOCKERBUILD" == 1 ]] && update-rc.d letsencrypt enable
```

, so it only gets called in the docker case

Reply:
No, that's wrong.
The `configure()` get's called when you enable letsencrypt on the web interface.
During build only the `install()` will be called.

With this change, you can never enable letsencrypt!

Reply:
that variable is `1`, because I used `ENV` in the dockerfile.

you can open a shell with `docker exec -ti bash nextcloudpi` and verify it

Reply:
You are totally right. I missed that.
You're proposal works fine. Thanks for catching that.